src/Widgets/TimerTimeWidget.vala

```diff
@@ -99,6 +99,7 @@ namespace Hourglass.Widgets {
             chooser_grid.attach (new Spacer.w_hexpand (), 0, 0, 1, 1);
             chooser_grid.attach (hour_chooser, 1, 0, 1, 1);
 
+            // TRANSLATORS: Time separator, e.g. 00:00:00
```

Comment:
```suggestion
            //TRANSLATORS: Time separator, e.g. 00:00:00
```

Does this work?

Reply:
@ryonakano unfortunately not, every comment is still extracted. Don't know why :confused:

Reply:
Hmm, strange, this removes all unwanted comments on my side :thinking:

Reply:
@ryonakano fixed now. I had to remove the colon you made me add in the previous suggestion AND to apply this suggestion too.

I also reconfigured the build before trying again, maybe there was a sort of cache issue.